.github/workflows/ci.yml

```diff
@@ -120,17 +123,68 @@ jobs:
         run: |
           ddev composer install
           ddev pull platform -y
-          ddev drush deploy -y
+
+      - name: Run phpstan
+        run: ddev php ./vendor/bin/grumphp run --tasks=phpstan --no-interaction
```

Comment:
@zeshanziya, I see you have made this to the top, and that makes sense, but let's do that in another PR? I want this PR to be an example of change we can use elsewhere as well.

Reply:
Noted